### PR TITLE
Add `make lint-fix-fieldalignment` 

### PR DESCRIPTION
**Pull request by andreafalzetti**
# Proposed changes

Now that we use the linter `fieldalignment` it's handy to have this make command to remember how to address fieldalignment issues.

Using `golangci-lint run --fix` does not seem to work.

## How to validate

1. If you already have `fieldalignment` installed remove it
1. Make a change in a struct causing a lint issue for `fieldalignment`
1. Run `make lint-fix-fieldalignment`
1. Notice how `fieldalignment` is installed and it fixed the issues. It will output the errors that it found.

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

**Comment by AdrianPedriza** [comment]
if I run `make lint-fix-fieldaligment` I get:
Captura de pantalla 2023-11-22 a las 10 44 09
It is expected?

**Comment by andreafalzetti** [comment]
> if I run `make lint-fix-fieldaligment` I get: Captura de pantalla 2023-11-22 a las 10 44 09 It is expected?

@AdrianPedriza Does it install the bin in `~/go/bin`?

**Review by teresaromero — APPROVED** [review]

**Comment by AdrianPedriza** [comment]
> ~/go/bin

yes, it does

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,22 @@ lint:
 	pre-commit run --all-files
 	golangci-lint run
 
+.PHONY: install-fieldalignment
+install-fieldalignment:
```
Comment (andreafalzetti) [inline Makefile:39]: I've made this step so it can be reused by the other two steps below.

**Review by AdrianPedriza — APPROVED** [review]

**Pull request merged**